components/script/test.rs

```diff
@@ -15,6 +15,33 @@ pub mod area {
     pub use dom::htmlareaelement::{Area, Shape};
 }
 
+pub mod sizes {
+    use app_units::Au;
+    use style::media_queries::{MediaQuery, MediaQueryType};
+    use style::media_queries::Expression;
+    use style::servo::media_queries::{ExpressionKind, Range};
+    use style::values::specified::{Length, NoCalcLength, AbsoluteLength};
+
+    pub use dom::htmlimageelement::{parse_a_sizes_attribute, Size};
+
+    pub fn test_media_query(len: f32) -> MediaQuery {
```

Comment:
These functions should live in tests/unit/script/htmlimageelement.rs instead of here.

Reply:
Adding this function giving lot of errors https://gist.github.com/Rakhisharma/a28c6a01f3e3a0b6b49fe90948e9fabe  Do you still want me to refactor this function in tests/unit/script/htmlimageelement.rs ?

Reply:
Right, you need to change the use statements to reference the new location instead of `script::test`.

Reply:
And when I said these functions, I meant test_media_query and test_length in particular.